content/post/RLadies_video_2019.Rmd

```diff
@@ -0,0 +1,227 @@
+                               'WomenRLadies', 'Rstn_RLadies13', 'RnRladies')) %>%
+  # We keep these three variables that allow us to identify each account
+  # with the number of followers each one has
+  select(screen_name, location, followers_count)
+
+# We calculate the total number of followers for all accounts
+rladies %>% 
+  summarise(sum(followers_count))
+
+```
+
+The image selected for this part of the video was taken in LatinR 2019. We were preparing for the R-Ladies group photo and, without realizing it, we formed a heart!! (which was captured by [TuQmano´s](https://twitter.com/TuQmano) eye and camera). The image represents the growth of R-Ladies, also beyond North-America and Europe, and the code that moves us to work as a team for the well-being of R-Ladies and the community in general.
+
+![](../image/corazon.png)
+
+
+The final text of the scene was:
+
+_We have more than 65,000 followers in our Twitter accounts, 940 experts in the R-Ladies directory, 80 international reviewers in our review network and we produce more than 600 documents with teaching materials. All done with 100% volunteer work_
+
+
+### Scene three: good wishes!
+
+Here we wanted to wish a happy year for all R-Ladies, and also for all the allies who accompanied us during the year. The selected image is our logo and our wesite address.
+
+The text for this scene is:
+
+_Happy New Year to all R-Ladies and allies! More information at rladies dot org_
+
+![](../image/placafinal.png) 
+
+
+### Language
+
+R-Ladies is a global community, the video had to be in English, the language the world speaks. But why not also in Spanish? The R Latin-American Community has grown a lot during this time and it has been largely thanks to the effort and work of the R-Ladies in this region of the world. Thus, we decided to generate the video in both languages to celebrate this hard work. [Laura Acion](https://twitter.com/_lacion_/) was responsible for correcting and translating the text of each scene.
+
+### Text, images ... audio?
+
+Now, a video with only letters, numbers, and images would leave many people out of our message, so we decided to record the audio of the message. For that, we got the help from the genius of [Alejandra Bellini](https://twitter.com/AlejaBellini), who recorded the audio in Spanish and English. She recorded it using WhatsApp with a cell phone, then we used [Zamzar](https://www.zamzar.com) to transform the audio to MP3 and [Mp3cut](https://mp3cut.net/en/) to cut the audio in the parts necessary to be able to synchronize the audio with the text and the video images.
+
+
+## Third step: editing ...
+
+_Third step_: with this plan in mind, it was time to edit the video. We did it using Doodly for the drawing-hand effects. Both music and fonts are provided by the software. The most laborious part was the synchronization of the audio with the drawing of the numbers and letters.
```

Comment:
We used the software Doodly, providing music and fonts, for the drawing-hand effects.